Source/Core/DiscIO/DirectoryBlob.h

```diff
@@ -307,7 +307,7 @@ class DirectoryBlobReader final : public BlobReader
   void SetPartitions(std::vector<PartitionWithType>&& partitions);
   void SetPartitionHeader(DirectoryBlobPartition* partition, u64 partition_address);
 
-  VolumeDisc* GetWrappedVolume() { return m_wrapped_volume.get(); }
+  VolumeDisc* GetWrappedVolume() const { return m_wrapped_volume.get(); }
```

Comment:
If you're going to make this member function `const`, it should probably return a `const VolumeDisc*`.